zerver/management/commands/deliver_email.py

```diff
@@ -36,6 +36,12 @@
 logger.setLevel(logging.DEBUG)
 logger.addHandler(file_handler)
 
+try:
```

Comment:
Our use of `mailer` is deprecated, and can be removed (from here and anywhere else).